lib/msf/base/serializer/readable_text.rb

```diff
@@ -859,8 +860,10 @@ def self.dump_sessions(framework, opts={})
           'Header' => "Active sessions",
           'Columns' => columns,
           'Indent' => indent)
-      framework.sessions.each_sorted { |k|
-        session = framework.sessions[k]
+
+      framework.sessions.each { |k|
```

Comment:
I imagine we'll still want `each_sorted` here, not sure if it was dropped intentionally or not 👀

Reply:
It didn't seem necessary and was causing issues with testing, but I'll fix it 💪